File: src/main/java/com/connectflow/dto/CustomerDTO.java
package com.connectflow.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;
import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CustomerDTO {
    private UUID id;
    private String fullName;
    private String nic;
    private String phone;
    private String address;
    private String customerType;
    private Boolean isActive;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: next/backend/src/dto/CustomerDTO.js
'use strict';
class CustomerDTO {
  constructor(customer) {
    this.id = customer.id;
    this.fullName = customer.fullName;
    this.nic = customer.nic;
    this.phone = customer.phone;
    this.address = customer.address;
    this.gender = customer.gender;
    this.customerType = customer.customerType;
    this.isActive = customer.isActive;
    this.createdAt = customer.created_at || customer.createdAt;
    this.updatedAt = customer.updated_at || customer.updatedAt;
  }
}
module.exports = CustomerDTO;
